provision/README.md

```diff
@@ -0,0 +1,22 @@
+# Provision
+
+## Overview
+
+The `hydroform` package contains the library used to create and manage clusters.
```

Comment:
```suggestion
The `provision` package contains the library used to create and manage clusters.
```
Shouldn't it say `provision`? Since we divided the main hydroform package into provision and install :)

Reply:
It should still say 'hydroform' here, as even though we have moved the package to a new directory, the name of the package in hydroform.go is still the same.